src/routes/auth/login/index.ts

```diff
@@ -20,12 +20,57 @@ export const post: RequestHandler = async (request) => {
 	};
 
 	if (!account) {
-		return {
-			status: 401,
-			body: {
-				errors: 'USER_NOT_FOUND',
-			},
-		};
+		account =
+			((await knex('account')
```

Comment:
we could use [Promise.any()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/any)

Reply:
Ideally, we would use a proper query, but I couldn't make it work with joins =/ Would you know how to?